### ISSUE TITLE
[Feature] Use Image with Idle Label

### ISSUE THREAD

**Issue by SamratSahoo** [pqina/vue-filepond#260]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

When I make upload boxes, I generally think it would be nice to have an image that I could put next to my Idle label to indicate that this is an upload box and to make it more aesthetically pleasing ([for example](https://ibb.co/9v9d3LM))

### Describe the solution you'd like

I think it would be nice if we could have some sort of prop that allows us to specify our idle image label as a property
```
<file-pond
      label-idle="Drag a File or Browse to Upload an Image"
      image-label-idle="idle.png"
    />
```

### Describe alternatives you've considered

- Manually creating an upload box
- Using a different library that may support something similar

**Comment by maverickchangithub** [comment]
yes you can . but not in your way , label-idle request a string , then turn into html.
you can add
```
label-idle='<img src="img-path" />'
```

remember to resize the image or it will be bleeding out the block
